modules/performing-an-image-signature-verification.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * operating/verify-image-signatures.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="performing-an-image-signature-verification_{context}"]
+= Performing an image signature verification
+
+Before you perform an image signature verification, you must first create a signature integration in {rh-rhacs-first}.
+
+When you create a signature integration, you can use the following verification methods:
```

Comment:
These steps should be in a separate module, "Creating a signature integration." If it already exists, you can xref to it in the assembly after this module.